packages/graphiql-react/src/editor/context.tsx

```diff
@@ -169,21 +169,25 @@ export type EditorContextProviderProps = {
    * typing in the editor.
    */
   headers?: string;
+  /**
+   * @deprecated Use `defaultTabs` instead.
+   */
+  initialTabs?: TabDefinition[];
   /**
    * This prop can be used to defined the initial set of tabs with their queries,
```

Comment:
Agreed that `defaultTabs` is more appropriate than `initialTabs`. Can we make the comment more clear? From your PR comment: "This prop is used to define the default set of tabs, with their queries, variables, and headers, that will be shown if there is no tab state persisted in localStorage".

Reply:
Definitely! Adjusted the text 👍